spec/system/support_interface/authentication_spec.rb

```diff
@@ -38,9 +42,11 @@ def then_i_should_be_redirected_to_login_page
   end
 
   def and_i_should_not_see_support_menu
-    expect(page).not_to have_link 'Applications'
-    expect(page).not_to have_link 'APITokens'
-    expect(page).not_to have_link 'Vendors'
+    expect(page).not_to have_link 'Candidates'
+    expect(page).not_to have_link 'Providers'
+    expect(page).not_to have_link 'Performance'
+    expect(page).not_to have_link 'Settings'
+    expect(page).not_to have_link 'Documentation'
   end
```

Comment:
More of a stylistic thing, but this is the reason I tend to prefer testing that you're on the correct page i.e.

expect(page).to have_current_path candidate_interface_sign_in_page 

When you only test something is present on the page, you could just be on the wrong page and the test will still pass. 

p.s. I get you're fixing a pre-existing test and it wasn't your implementation